tests/data/binomial_data.py

```diff
@@ -0,0 +1,147 @@
+import random
+
+import geomstats.backend as gs
+from geomstats.information_geometry.binomial import BinomialDistributions
+from tests.data_generation import _OpenSetTestData
+
+
+class BinomialTestData(_OpenSetTestData):
+    space = BinomialDistributions
+    n_list = random.sample((2, 5), 1)
+    n_samples_list = random.sample(range(10), 3)
+    space_args_list = [(n,) for n in n_list]
+    shape_list = [(1,)]
+    n_points_list = random.sample(range(5), 2)
+    n_vecs_list = random.sample(range(2, 5), 1)
+
+    def belongs_test_data(self):
+        smoke_data = [
+            dict(n_draws=10, vec=0.1, expected=True),
+            dict(n_draws=8, vec=-0.8, expected=False),
+            dict(n_draws=5, vec=0.1, expected=True),
+            dict(n_draws=2, vec=-1.0, expected=False),
```

Comment:
- `vec` is misleading as your are testing a point on the manifold and not a tangent vector. 
- Values higher than 1 are not tested.
- For one point, maybe test `gs.array([x])` as well as float `x`.
- Several points are not tested, i.e. `gs.array([x, y])` as well as `[x, y]` and `gs.array([[x],[y]])`.

Reply:
`vec` -> `point`